src/services/user-service.spec.js

```diff
@@ -52,6 +52,78 @@ describe('user-service', () => {
     sandbox.restore();
   });
 
+  describe('findConfirmedActiveUserById', () => {
```

Comment:
We could still throw the `expires[On]` case in here, in both methods 😅

Reply:
I'm not sure what you mean :thinking:

Reply:
The method has this part that created the bug fixed today:
```
   if (!user || user.expiresOn) {
      return null;
    }
```
with `expires` vs `expiresOn` (sorry I made it look like an array item accessor with `expires[On]`, I meant +/- the `On` part)

Reply:
I'm not sure if we're already taking about the same thing. But it's a good idea to add test cases for both `findConfirmedActiveUserById` and `findConfirmedActiveUserByEmailAndPassword` that check that no user with `expiresOn` is let through by the query. I'll do that :+1:

Reply:
I have added those test cases.

Reply:
Perfect! 👌